src/graphql/gh.graphql

```diff
@@ -0,0 +1,25 @@
+query orgs {
```

Comment:
@0xtsukino 
This can return the list of committers of a given repo in a given timeframe (I still need to use the since/until parameters).

It seems it is not possible to queries multiple organizations at once though.
https://github.com/orgs/community/discussions/24590#discussioncomment-3244575

So it means if if use 100 orgs >> 100 requests.
I haven't tested if it hits a rate limit.
You said we would do this job only once per month right?

Reply:
nice!

we can run this around once per month (say 1200 UTC on the first of every month), and store all committers to postgres. we can then allow user to join group for 3 months.

so we should also stored the `since/until` parameters so we have data to calculate ttl of each commitment.